### PR TITLE
Fix for udemy.com

### PR DESCRIPTION
**Pull request by felipebarbosaferreira**
The practice questions radio buttons were the same color even when selected. Now when selected it has highlighted color ;)

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -13725,6 +13725,15 @@ g#ubuntu-logo > path
 
 udemy.com
 
+CSS
+.radio:not(.radio-button) input[type="radio"]:checked + .radio-label::before, .radio-inline:not(.radio-button) input[type="radio"]:checked + .radio-label::before {
+    background-color: rgb(93, 122, 136);
+    box-shadow: rgb(27, 29, 29) 0px 0px 0px 2px inset;
+}
+div[class^='curriculum-item-view--scaled-height-limiter'], div[class*=' curriculum-item-view--scaled-height-limiter'] {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13733]: Ehm, we should dark reader change the max-height?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -13725,6 +13725,15 @@ g#ubuntu-logo > path
 
 udemy.com
 
+CSS
+.radio:not(.radio-button) input[type="radio"]:checked + .radio-label::before, .radio-inline:not(.radio-button) input[type="radio"]:checked + .radio-label::before {
+    background-color: rgb(93, 122, 136);
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13730]:
```suggestion
    background-color: ${#6f92a3} !important;
```
What about changing it a little bit.

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -13725,6 +13725,15 @@ g#ubuntu-logo > path
...
+    box-shadow: rgb(27, 29, 29) 0px 0px 0px 2px inset;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13731]: Is the box-shadow necessary to be modified?

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -13725,6 +13725,15 @@ g#ubuntu-logo > path
 
 udemy.com
 
+CSS
+.radio:not(.radio-button) input[type="radio"]:checked + .radio-label::before, .radio-inline:not(.radio-button) input[type="radio"]:checked + .radio-label::before {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13729]:
```suggestion
.radio:not(.radio-button) input[type="radio"]:checked + .radio-label::before,
.radio-inline:not(.radio-button) input[type="radio"]:checked + .radio-label::before {
```

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (felipebarbosaferreira) [reply src/config/dynamic-theme-fixes.config:13733]: my 2560x1080 (ultra wide) screen

### without the change

Screenshot_20211105_092014


### with the change, the space of the content is for the questions is proportional to the screen 

Screenshot_20211105_091926

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:13733]: Is that a bug that Dark Reader caused? Otherwise it isn't a fix, just a cosmetic change.

Comment (felipebarbosaferreira) [reply src/config/dynamic-theme-fixes.config:13733]: ```we should dark reader change the max-height?```


for me the change makes sense, but I think the site that should fit my screen

dark reader is more for theme, not layout. but it has helped me a lot at this point

the decision is yours ;)

if not added by you, I'll have to keep putting it in the dynamic theme every time (it's not saving the settings)

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (felipebarbosaferreira) [reply src/config/dynamic-theme-fixes.config:13730]: I agree \o/

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (felipebarbosaferreira) [reply src/config/dynamic-theme-fixes.config:13731]: it's not necessary, I was trying to make it smoother

File: src/config/dynamic-theme-fixes.config (hunk 4 above)
Comment (felipebarbosaferreira) [reply src/config/dynamic-theme-fixes.config:13729]: agree, clean look

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (felipebarbosaferreira) [reply src/config/dynamic-theme-fixes.config:13733]: > Is that a bug that Dark Reader caused? Otherwise it isn't a fix, just a cosmetic change.

not a bug

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:13733]: > for me the change makes sense, but I think the site that should fit my screen

I get that, but this config is exclusively for fixes that is caused by Dark Reader.

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:13731]: I consider it as a cosmetic change and not necessary a fix.

**Pull request closed without merging**